### ISSUE TITLE
[Regression] Current line highlight transparency not working

### ISSUE THREAD

**Issue by GoblinAlchem** [notepad-plus-plus/notepad-plus-plus#5619]
### Description of the Issue

The current line highlighting feature is useful, but it becomes misleading if background color is also used in syntax highlighting. A compromise is to use semi-transparent highlighting, or assign an alpha value to the current line background color. To do this, I have found the following instruction:

http://docs.notepad-plus-plus.org/index.php/Current_Line_Highlighting

I have created a macro which uses the "wParam" parameter to set the alpha value:

`<Macro name="Current line highlight 10%" Ctrl="yes" Alt="no" Shift="yes" Key="72">`
`    <Action type="0" message="2470" wParam="26" lParam="0" sParam="" />`
`</Macro>`

Since Notepad++ 7.6.4, this macro doesn't work.

### Steps to Reproduce the Issue

1. Add the above macro to shortcuts.xml
2. Start Notepad++
3. In Settings > Style Configurator > Global Styles > Current Line Background, set a contrasting color, return to editing mode
4. Press Ctrl+Shift+H

### Expected Behavior

The current line highlighting should become pale (semi-transparent)

### Actual Behavior

The current line highlighting remains the fully opaque color as specified in settings

### Debug Information

Notepad++ v7.6.6   (32-bit)
Build time : Apr  3 2019 - 23:49:50
Path : C:\Users\bederov\Programs\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 7 (64-bit)
Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll NppExport.dll 

### Previous version

Uninstalled the current version and installed 7.6.3, pressing Ctrl+Shift+H makes the current line highlighting semi-transparent.

Notepad++ v7.6.3   (32-bit)
Build time : Jan 27 2019 - 17:20:30
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 7 (64-bit)
Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll NppExport.dll

**Comment by sasumner** [comment]
Interesting.  It works via NppExec using `SCI_Sendmsg 2470 26 0` or via Pythonscript using `editor.setCaretLineBackAlpha(26)`, but the OP is correct, with the macro he shows it isn't working in N++ 7.6.6 when I try it.

BTW, when it does work, I'm not sure I see the usefulness of it.  To each his own, I guess.

**Comment by GoblinAlchem** [comment]
@sasumner, I am using this feature to have the current line color blended with other background colors, e.g. string literals. This macro is from [Notepad++ wiki](http://docs.notepad-plus-plus.org/index.php/Current_Line_Highlighting). If there is another way to make the current line color semi-transparent, I will be happy to use it, but the wiki will have to be corrected as well.

I have assigned the macro to a hotkey and I am pressing this hotkey when starting Notepad++. So I believe semi-transparent current line background should be available in settings. I have posted a feature request for that (#5620).
